Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM ubuntu:14.04
 MAINTAINER PhenoMeNal-H2020 Project <phenomenal-h2020-users@googlegroups.com>
 
-LABEL Description="Galaxy 17.09-phenomenal for running inside Kubernetes."
+LABEL Description="Galaxy 18.01-phenomenal for running inside Kubernetes."
 LABEL software="Galaxy"
-LABEL software.version="17.09-pheno-lr"
-LABEL version="1.6"
+LABEL software.version="18.01-pheno-isa"
```

Comment:
Can we get rid of such suffixes ? plain 18.01-pheno ?

Reply:
Would be great to have versioning in Dockerfile and Dockerfile_init in sync.

Reply:
Agreed!

Reply:
The suffixes at the end we mostly use to signal that there is stuff that is not part of the Galaxy release that we are using, like ISA support or restart improvements for k8s. As we progress in the Galaxy releases, those leave as they are part of the official Galaxy. In any case, its mostly only us that see this.